src/api.v2/controllers/sampleFileController.js

```diff
@@ -0,0 +1,55 @@
+const sqlClient = require('../../sql/sqlClient');
+
+const Sample = require('../model/Sample');
+const SampleFile = require('../model/SampleFile');
+
+const { getSampleFileUploadUrl } = require('../helpers/s3/signedUrl');
+const { OK } = require('../../utils/responses');
+const getLogger = require('../../utils/getLogger');
+
+const logger = getLogger('[SampleFileController] - ');
+
+const createFile = async (req, res) => {
+  const {
+    params: { experimentId, sampleId, sampleFileType },
+    body: { sampleFileId, size, metadata = {} },
+  } = req;
+  logger.log(`Creating sample file for experiment ${experimentId}, sample ${sampleId}, sampleFileType ${sampleFileType}`);
```

Comment:
Maybe `Creating file {sampleFileType} for sample {sample} in experiment {experimentId}`